### ISSUE TITLE
Getting an error: Filename cannot start with a dot

### ISSUE THREAD

**Issue by schneems** [esm7/obsidian-vimrc-support#173]
**Please make sure your question is within the scope of this plugin.**
- [x] I know this plugin is about *persisting changes* to the Vim support built into Obsidian, plus some bonus features, and considered asking in the [Obsidian forum](https://forum.obsidian.md/c/get-help/19) first.

**Describe the question**

Following the README, I tried creating a file and get this error:

<img width="558" alt="Screenshot 2023-03-23 at 11 56 47 AM" src="https://user-images.githubusercontent.com/59744/227278933-aa6c6d77-6d9f-4e16-83a2-aca6acdc7ee2.png">

Sorry for the bad screenshot, but the error goes away really fast so it's hard to trigger it and capture at the same time.

I'm relatively new to obsidian so it's likely I'm doing something silly.

**Comment by tjex** [comment]
yes, it's correct that you need to make the file outside of vim. 
I guess the maintainers overlooked this little detail, as those coming from vim / neovim will know this as second nature (creating and using dotfiles)

**Comment by schneems** [comment]
>  I'm guessing this is why obsidian will block you from making a file that starts with a period.

That was my guess as well. It makes sense. I'm also guessing/wondering if this behavior used to be possible (editing that file from within the GUI) but changed after an update to the app.

> as those coming from vim / neovim will know this as second nature (creating and using dotfiles)

I've used (terminal-based) vim as my full-time editor for years. I maintain dotfiles and sync them to a public GitHub repo https://github.com/schneems/dotfiles/tree/main/vim. I'm familiar with these concepts, but in the context of following instructions, I think it's worth spelling it out a little more.

Part of the confusion (for me) is that "file" means something in the context of Obsidian GUI, and since this is an obsidian plugin for the GUI, I assumed that "obsidian file" was meant to be set in the GUI. I thought that I would be able to edit my config directly in obsidian, much like I'm able to use VSCode to edit its own user json settings (or use vim to edit vimrc settings).

Also worth noting that I'm new to obsidian and didn't realize (until this response to you) that the folder on disk where I mentioned putting the file maps literally 1:1 to the notes in my vault.

I'm mentioning these things because it can be hard to break out of the curse of knowledge of being really familiar with a tool to understand how someone new to it (even if they're very competent in other, related areas) might misunderstand things in new and exciting ways. Also, developers tend to be shy about mistakes they make and internalize and hide them. I make a ton of mistakes all the time, I hope by sharing my issues, I can help others having problems and talk about ways to update systems to better handle failure modes and be kinder to beginners.

**Comment by tjex** [comment]
Well, then I'm sure the maintainer/s would appreciate a pr then to include the extra clarification for setup.

**Comment by Wastack** [comment]
You can also circumvent the issue by going Settings -> Community plugins -> Vimrc Support and change Vimrc file name to something that has no dot `.` at the beginning.
